components/brave_wallet_ui/panel/container.tsx

```diff
@@ -259,11 +261,16 @@ function Container () {
     }
   }
 
-  const onSelectTransaction = (transaction: BraveWallet.TransactionInfo) => {
+  const viewTransactionDetail = (transaction: BraveWallet.TransactionInfo) => {
     dispatch(WalletPanelActions.setSelectedTransaction(transaction))
     dispatch(WalletPanelActions.navigateTo('transactionDetails'))
   }
 
+  const viewTransactionStatus = (transaction: BraveWallet.TransactionInfo) => {
+    dispatch(WalletPanelActions.setSelectedTransaction(transaction))
+    dispatch(WalletPanelActions.navigateTo('transactionStatus'))
+  }
+
   const onConfirmTransaction = () => {
```

Comment:
Should this be a `useCallback`?

Reply:
Added `useCallback` in the methods changed by this PR.